lxc/config_device.go

```diff
@@ -244,7 +244,13 @@ func (c *cmdConfigDeviceGet) Run(cmd *cobra.Command, args []string) error {
 
 		dev, ok := inst.Devices[devname]
 		if !ok {
-			return fmt.Errorf(i18n.G("The device doesn't exist"))
+			_, ok = inst.ExpandedDevices[devname]
+			if !ok {
+				return fmt.Errorf(i18n.G("The device doesn't exist"))
+			}
+
+			return fmt.Errorf(i18n.G("Cannot get information on profile device, use `lxc profile device` instead"))
```

Comment:
@monstermunchkin thanks for taking a look at this :)

I know that @stgraber has mentioned to me in the past that we don't include references to the `lxc` commands in our API error messages, as it could be that the client is consuming the API directly and not using the `lxc` command.

So how about this: "The device exists in the instance's profile(s)"